src/components/tasks/card/TaskTagEdit.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+
+import classNames from '@/components/tasks/card/card.module.scss';
+import { useTasksContext } from '@/context/tasks.context';
+
+type TaskTagPropsType = {
+    updateTaskTagLevel: (taskItemToUpdate: any, method: 'delete' | 'post') => void
```

Comment:
Have we not created an interface for `Task` which could be utilized for `taskItemToUpdate` here right?

Reply:
`task` is a separate data object stored in different collection in backend, where as taskItem is stored in `items` collection,which has `itemType` as `TASK`